inginious/frontend/user_manager.py

```diff
@@ -345,6 +345,14 @@ def get_user_email(self, username):
             return info[1]
         return None
 
+    def has_accept_tos(self, username):
```

Comment:
You directly send a request to mongodb in ``utils.py``. It would be better to make a choice between the two ways to avoid duplicate code.